hw3/scripts/Homework-3_Group-Analysis.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Part 0: Prep
+3dmask_tool \
+-input masks/full_mask.sub-*+tlrc.HEAD \
+-frac 1.0 \
+-prefix full_mask.group
+
+
+# Part 1: Group statistics
+3dttest++ -mask full_mask.group+tlrc -prefix Words.group -setA 'betas/stats*+tlrc.HEAD[Words#0_Coef]'
+
+
+# Part 2: Cluster correction
+## Estimate Data Smoothness
+for f in blurs/blur_est.*.1D; do
+	1d_tool.py -infile $f -select_rows 3 -write - >> blurs.acf.1D
+done
+
+1dsum -mean blurs.acf.1D > blurs.acf.avg.1D
+
+3dClustSim \
+-mask full_mask.group+tlrc \
+-acf `1dcat blurs.acf.avg.1D'[0:2]'` \
+-both \
+-pthr .05 .01 .001 \
+-athr .1 .05 .025 .01 \
+-iter 2000 \
```

Comment:
This is ok for HW and in class, but use more iterations (at least 5000, preferrably the default of 10,000) for any real analysis.